spec/features/create_campaign_spec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "Create campaign" do
+  before do
+    given_i_am_signed_in
+    and_vaccines_exist
+  end
+
+  scenario "User creates a Flu campaign" do
+    when_i_go_to_the_campaigns_page
+    and_i_click_on_the_new_campaign_button
+    then_i_should_see_the_details_page
+
+    when_i_fill_in_flu_details
+    and_i_click_continue
+    then_i_should_see_the_dates_page
+
+    when_i_fill_in_the_dates
+    and_i_click_continue
+    then_i_should_see_the_confirm_page
+
+    when_i_confirm_the_campaign
+    then_i_should_see_the_campaigns_page
+    and_i_should_see_the_flu_campaign
```

Comment:
I wonder whether it's worth adding coverage here (and for the HPV scenario) that only active vaccines are added to the campaign?

Reply:
Yeah I think it's probably a good idea, I'll amend it.